packages/block-editor/src/components/iframe/index.js

```diff
@@ -49,6 +49,13 @@ function styleSheetsCompat( doc ) {
 			return;
 		}
 
+		// Don't try to add the reset styles, which were removed as a dependency
+		// from `edit-blocks` for the iframe since we don't need to reset admin
+		// styles.
+		if ( ownerNode.id === 'wp-reset-editor-styles-css' ) {
```

Comment:
We need to ignore this in the compatibility layer because it will search stylesheet for relevant selectors, which also matches `wp-reset-editor-styles`.